gammapy/obs/observation.py

```diff
@@ -38,24 +41,28 @@ class ObservationTable(Table):
     """Observation table (a.k.a. run list).
 
     This is an `~astropy.table.Table` sub-class, with a few
-    convenience methods and the following columns:
-
-    * ``OBS_ID``
-    * ``ONTIME``
-    * ``LIVETIME``
-    * ...
-
+    convenience methods. The format of the observation table
+    is described in :ref:`dataformats_observation_lists`.
```

Comment:
Yes, this is nice. One can even cross-link to Astropy or other doc sets via intersphinx.
I didn't know this a year ago, so there's plenty of places in Gammapy where one could cross-link high-level and API docs. Please do when you see opportunities for improvement or centralising duplicated documentation.